app/models/users.py

```diff
@@ -0,0 +1,55 @@
+from flask import Flask
+from marshmallow import Schema, fields
+from passlib.apps import custom_app_context as pwd_context
+import random
+from itsdangerous import (TimedJSONWebSignatureSerializer
+                          as Serializer, BadSignature, SignatureExpired)
+
+
+app = Flask(__name__)
```

Comment:
Any reason why you have initialized the app twice, here and in the views.py?

Reply:
It was giving an import error when I tried to import the app from views.py but when I initialized it here it worked fine.